homeassistant/components/bom/sensor.py

```diff
@@ -73,7 +74,7 @@
     "rel_hum": ["Relative Humidity", UNIT_PERCENTAGE],
     "sea_state": ["Sea State", None],
     "swell_dir_worded": ["Swell Direction", None],
-    "swell_height": ["Swell Height", "m"],
+    "swell_height": ["Swell Height", LENGTH_METERS],
```

Comment:
@maddenp are you talking about this instance?

Reply:
Yes, but also the `"Precip Accumulation"` block in `darksky/sensor.py` -- accumulated precipitation is thought of in a vertical context. Having both length- and height-context constants might allow maintainers to choose the constant that best fits their application. I don't feel strongly about it, but if there are no obvious downsides, it might make for more expressive code.

Reply:
IMHO unit of measurement should not take direction into account, because it is only meant to describe in what kind of unit something is measured or displayed e.g. in this case meters. Moreover, adding another constant that brings clarity but describes the same value is not meaningful.

Reply:
A rhetorical question: Doe the `LENGTH_` prefix add anything? What else could `METERS` be describing? Not having the `LENGTH_` prefix would avoid the direction question altogether. Just food for thought, I'm not actually suggesting a change.

Reply:
Oh okay, these are actually good questions. I believe the prefix is a good way of grouping the various constants, so it is easier to differenciate between the long list of constants.